DLaB.Xrm.Tests.Core/FakeContext.cs

```diff
@@ -70,6 +70,7 @@ public Entity? Target
         public IOrganizationService SystemOrganizationService { get; set; } = null!;
         public FakeTraceService FakeTraceService { get; set; } = new FakeTraceService(new DebugLogger());
         public ITracingService TracingService => FakeTraceService;
+        public IManagedIdentityService ManagedIdentityService { get; set; } = null!;
```

Comment:
Would it make sense to create a FakeManagedIdentityService in the XrmUnitTest that we could use here?

Reply:
I put in a pull request to add in a new FakeManagedIdentityService, that would need to be implemented before I can do that correct?